### ISSUE TITLE
no method matching `setup_call(::LoopVectorization.LoopSet, ::Expr, ::LineNumberNode, ::Bool, ::Bool, ::Int8, ::Int8, ::Int64)`

### ISSUE THREAD

**Issue by ranocha** [JuliaSIMD/LoopVectorization.jl#297]
We get the following error from GitHub CI in Trixi.jl:
```
  LoadError: MethodError: no method matching setup_call(::LoopVectorization.LoopSet, ::Expr, ::LineNumberNode, ::Bool, ::Bool, ::Int8, ::Int8, ::Int64)
  Closest candidates are:
    setup_call(::LoopVectorization.LoopSet, ::Expr, ::LineNumberNode, ::Bool, ::Bool, ::Int8, ::Int8, ::Int64, ::Bool) at C:\Users\runneradmin\.julia\packages\LoopVectorization\PDSoT\src\condense_loopset.jl:790
  Stacktrace:
    [1] #s96#15
      @ C:\Users\runneradmin\.julia\packages\StrideArrays\NCjmP\src\broadcast.jl:345 [inlined]
    [2] var"#s96#15"(S::Any, D::Any, T::Any, N::Any, C::Any, B::Any, R::Any, BC::Any, UNROLL::Any, ::Any, dest::Any, bc::Any, #unused#::Any)
      @ StrideArrays .\none:0
    [3] (::Core.GeneratedFunctionStub)(::Any, ::Vararg{Any, N} where N)
      @ Core .\boot.jl:571
    [4] materialize!
      @ C:\Users\runneradmin\.julia\packages\StrideArrays\NCjmP\src\broadcast.jl:351 [inlined]
```
See https://github.com/trixi-framework/Trixi.jl/runs/3015829828?check_suite_focus=true#step:6:1811

**Comment by Cczernik** [comment]
I get the same error on Mac OS. 

However, when using ` pkg> add LoopVectorization@v0.12.49` everything works fine.

**Comment by chriselrod** [comment]
Should be fixed by
https://github.com/chriselrod/StrideArrays.jl/commit/264b054506a6b52b578f6f66f2d6c607b085087e

**Comment by ranocha** [comment]
Great! Thanks a lot for the quick fix! I guess it will be released in StrideArrays v0.1.11 soon?

**Comment by chriselrod** [comment]
> Great! Thanks a lot for the quick fix! I guess it will be released in StrideArrays v0.1.11 soon?

Ah, 0.1.11. Thanks for reminding me to bump the version. Not sure why 0.1.9 was listed as the latest release, even though 0.1.10 is also a tagged release: https://github.com/chriselrod/StrideArrays.jl/commit/75ffbcb821e0aacfb070c5f915e039f0c8784c3f

**Comment by ranocha** [comment]
Great, thanks :+1:
